i18n-util/src/LocaleUtil.js

```diff
@@ -3,8 +3,8 @@ const LOCALE_COOKIE_KEY = 'okta_help_user_lang';
 const getLocale = () => {
   // map to convert locales to folder names within h.o.c
   const supportedLocaleToFolderMap = {
-    'en-US': 'en',
-    'ja-JP': 'ja',
+    'en-US': 'en-us',
```

Comment:
We map the locale we compute to the folder names within h.o.c. The folder are now going to be `en-us`, `ja-jp` and so on.

Reply:
Keeping this map actually also helps us to also guard against an invalid locale that gets passed in (for end user docs). As per the current logic if we send in `en-JP` then when we do the look up on `supportedLocaleToFolderMap ` it will return undefined and then default to `en-us`.